src/components/Nav/NavSubmenuItem.react.js

```diff
@@ -17,11 +18,12 @@ function NavSubmenuItem({
   icon,
   children,
   active = false,
+  to,
 }: Props): React.Node {
   const classes = cn({ "nav-item": true, active: active }, className);
   const Component = RootComponent || "a";
   return (
-    <Component className={classes}>
+    <Component className={classes} to={to}>
```

Comment:
Every single `NavSubmenuItem` uses a `RootComponent` of `NavLink`.  Why not just automatically return `<NavLink className=...` and save ourselves the trouble of passing that prop every time?  It seems like we will always want the submenu items to be links.

Reply:
I've been trying to remember exactly why I did it this way but I can't at the moment, mind if I leave it like this just for the time being while I continue on the demo site and it should come to me and/or I'll amend this?

Reply:
Sounds good!  I just like avoiding <Component wherever possible because then we lose a lot of the benefits of flow because it's unable to check the props.